### PR TITLE
Support comparing ORC data

### PR DESCRIPTION
**Pull request by wjxiz1992**
Signed-off-by: Allen Xu <allxu@nvidia.com>

This is to resolve https://github.com/NVIDIA/spark-rapids/issues/1544

### CODE REVIEW

File: integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/CompareResults.scala [hunk 1]
```diff
@@ -53,6 +53,8 @@ object CompareResults {
         (spark.read.csv(conf.input1()), spark.read.csv(conf.input2()))
```
Comment (gerashegalov) [inline integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/CompareResults.scala:53]: Similarly lines https://github.com/NVIDIA/spark-rapids/pull/1545/files#diff-bc99ae208fb44d4abcf91b6e5b44515fbebeb90367c5f9eb1719e28226d22bf9R51-R68
can be 
```scala
    val format = spark.read.format(conf.inputFormat())
    BenchUtils.compareResults(
      format.load(conf.input1()),
      format.load(conf.input2()),
      conf.inputFormat(),
      conf.ignoreOrdering(),
      conf.useIterator(),
      conf.maxErrors(),
      conf.epsilon())
  }
```

**Review by gerashegalov — APPROVED** [review]
LGTM, nits

File: integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/BenchUtils.scala [hunk 2]
```diff
@@ -525,6 +525,8 @@ object BenchUtils {
         path: String => spark.read.csv(path)
       case "parquet" =>
         path: String => spark.read.parquet(path)
+      case "orc" =>
+        path: String => spark.read.orc(path)
```
[the remark this answers is not in the document]
Comment (wjxiz1992) [reply integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/BenchUtils.scala]: updated.

File: integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/CompareResults.scala (hunk 1 above)
Comment (wjxiz1992) [reply integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/CompareResults.scala]: updated.

File: integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/BenchUtils.scala [hunk 3]
```diff
@@ -520,12 +520,7 @@ object BenchUtils {
```
Comment (jlowe) [inline integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/BenchUtils.scala:520]: 2021 copyrights need to be added to this file.

File: integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/CompareResults.scala [hunk 4]
```diff
@@ -48,16 +48,11 @@ object CompareResults {
         .config("spark.rapids.sql.enabled", "false")
```
Comment (jlowe) [inline integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/CompareResults.scala:48]: 2021 copyrights need to be added to this file.

**Review by jlowe — CHANGES_REQUESTED** [review]

File: integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/BenchUtils.scala (hunk 3 above)
Comment (wjxiz1992) [reply integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/BenchUtils.scala:520]: added.

File: integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/CompareResults.scala (hunk 4 above)
Comment (wjxiz1992) [reply integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/CompareResults.scala:48]: added

**Review by jlowe — DISMISSED** [review]

**Comment by jlowe** [comment]
build

**Comment by wjxiz1992** [comment]
build

**Review by jlowe — APPROVED** [review]

**Review by gerashegalov — APPROVED** [review]
LGTM

**Pull request merged**